src/test/java/org/jenkinsci/plugins/gitclient/GitAPITestCase.java

```diff
@@ -4085,7 +4119,7 @@ public void test_autocreate_fails_on_multiple_matching_origins() throws Exceptio
         w.git.fetch_().from(remote, refspecs).execute();
 
         try {
-            w.git.checkout().ref(Constants.MASTER).execute();
+            w.git.checkout().ref("1.4.x").execute();
```

Comment:
This change hints that something has changed in an incompatible way.  I suspect that switching to "clone" instead of "init + fetch" means that a branch matching the remote default branch name (usually "master") is always created.  That will be a functional regression for users that depend on the initial checkout not creating a named branch unless specifically requested to do so.

Reply:
Yes, the default branch of the origin or the branch explicitly set using the -b option is implicitly available, when cloning a repository.